src/enr/enr.ts

```diff
@@ -182,7 +182,7 @@ export class ENR extends Map<ENRKey, ENRValue> {
   get tcp6(): number | undefined {
     const raw = this.get("tcp6");
     if (raw) {
-      return muConvert.toString(Multiaddr.protocols.names.tcp.code, raw) as number;
+      return muConvert.toString(protocols.names.tcp.code, raw) as number;
```

Comment:
Is the casting to number still necessary?

Reply:
There is a test failing that I could not understand why last night. I think it is related to this now that you point it out. It returns a [String](https://github.com/multiformats/js-multiaddr/blob/master/src/convert.js#L34). But there is these ]local definitions](https://github.com/ChainSafe/discv5/blob/9a336ebfe44068a11b1f54628fc9fdb74ee168d4/types/multiaddr/index.d.ts#L8) which also return a number

Reply:
fixed the local type definitions. we do not support types in multiaddr with an import path, as we do in libp2p interfaces. This is because we would not expect these utils to be used externally. Perhaps worth creating a follow up discussion on how we can enable this better, with a multiaddr utils, or exporting in the root